trailblazer/constants.py

```diff
@@ -41,6 +41,7 @@ class Workflow(StrEnum):
     MIP_DNA: str = "MIP-DNA"
     MIP_RNA: str = "MIP-RNA"
     MUTANT: str = "MUTANT"
+    RSYNC: str = "RSYNC"
```

Comment:
Should I refactor this to the "proper" ENUM handling in this PR @Vince-janv ?

Reply:
I say do it in another PR!

Reply:
I agree with SA. I think these  ENUM refactorings can be quite error-prone so doing them in separate PRs would be good IMO